mods/ts/rules/nod-vehicles.yaml

```diff
@@ -199,7 +201,7 @@ SAPC:
 		Cost: 800
 	Tooltip:
 		Name: Subterranean APC
-		Description: Armored infantry transport and mobile AA\n  Strong vs Aircraft, Vehicles\n  Weak vs Infantry
+		Description: Armored infantry transport and mobile AA.\nIs able to move underground.\n  Strong vs Aircraft, Vehicles\n  Weak vs Infantry
```

Comment:
Strong versus nothing, actually.
And I think it hardly qualifies as "armored".
How about "Troop transport that can move underground to avoid detection." ?